docs/Product-Guides/Benefits-API.md

```diff
@@ -1,40 +1,253 @@
+
+For most of your customers, this is already a part of the HR admin workflow. We are just changing the location of where they make this change. It is important as the developer to make it clear to your customers what the payroll close date is so that it can be communicated properly if the benefit will take effect this pay period or next pay period.
+
+## Example
+
+As an example, if the payroll period is **June 1 - 15**. The payroll close date might be **June 16** so payroll can be processed before **Friday, June 17**.  Therefore, it would be important to submit payroll deductions via the Finch API by **June 9** for those to take effect during the June 1 - 15 payroll.
+
+
+<pre>
+June 2022
+Su Mo Tu We Th Fr Sa
+          <mark style="background-color: lightblue">1  2  3  4
+ 5  6  7  8  9 10 11
+12 13 14 15</mark> <mark style="background-color: yellow">16</mark> <mark style="background-color: green">17</mark> 18
+19 20 21 22 23 24 25
+26 27 28 29 30
+</pre>
+
+### Couple of things to keep in mind
+
+- Often there is a company contribution and an employee contribution. Some payroll systems separate these, some add them together when calculating the deduction. When validating if the deduction is successful, make sure to understand how the provider you are working with handles contributions. Our team will add the deduction based on how the provider is set up.
+- Some payroll providers offer a dedicated payroll representative to help with making payroll changes. If a payroll rep is helping your customer’s HR admin with changes in their system, it is important that you make it explicitly clear who does what so that the payroll rep does not overwrite any changes Finch has previously made. Calling out Benefit Code types and thoughtful benefit descriptions help.
+
+## Testing benefits
+
+There are three ways we recommend testing benefits.
+
+### Option 1 - Sandbox
+
+You can [use the benefits endpoints in the Finch Sandbox](https://developer.tryfinch.com/docs/reference/ZG9jOjMxOTg1NTMw-testing#testing-benefits). This supports read operations (`GET` requests) and stubbed out write operations (`POST` requests), meaning writes will not actually change any internal state and reads will always return the same responses.
+
+### Option 2 - Automated
+
+If you need to test reading back the data you write into the system, you can use our test Gusto instance (using the company Cisticola LLC that we set up) for testing our benefits endpoints. At the time of writing (6/22/22), this Gusto instance currently has two active employees, one active contractor, and two inactive employees. The active employees can be enrolled in any of the benefits that Gusto supports. The compatibility matrices of support benefit types and benefit features can be found below - [Gusto Compatibility Matrices]().
```

Comment:
I think details like specific test accounts, numbers of employees, etc are better left out of our docs. I'd rather we ask them to contact us to get set up with a test account, and provide the details at that point.

Also generally good to remember that Merge is reading everything written in our docs